routers/web/repo/projects.go

```diff
@@ -385,9 +385,11 @@ func UpdateIssueProject(ctx *context.Context) {
 
 	projectID := ctx.FormInt64("id")
 	for _, issue := range issues {
-		oldProjectID := issue.Project.ID
-		if oldProjectID == projectID {
-			continue
+		if issue.Project != nil {
+			oldProjectID := issue.Project.ID
+			if oldProjectID == projectID {
```

Comment:
As above, why not just do
```suggestion
			if issue.Project.ID == projectID {
```